### DISCUSSION TITLE
Welcome to CTSM Discussions!

### DISCUSSION THREAD

**Discussion by billsacks** [ESCOMP/CTSM#1830] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wwieder** [comment]
Is this something we want to use?  How?  what about the forum?

**Reply by ekluzek** [reply]
We want to try this out. It makes it easier to move discussions into issues and vice versa. We will still answer on the CESM forums, but this will be better for closer integration with our github issues.

Opening something here that you want the CTSM team to look at is I think the preferred way of starting a discussion on something that will be visible to all. I think the forums come in especially for CESM public releases. But, users's can post either place. But, these should be preferred over private emails.

**Comment by ekluzek** [comment]
The land forum for CESM discussions is here by the way:

https://bb.cgd.ucar.edu/cesm/forums/ctsm-clm-mosart-rtm.134/

**Reply by wwieder** [reply]
+1 to @ekluzek post on CESM discussions,  
The advantage of these public discussions is that it's a way for you to read about suggestions for others in the same situation (and inform others who may have similar questions down the road).  [This board](https://bb.cgd.ucar.edu/cesm/forums/ctsm-clm-mosart-rtm.134/) is monitored by the LWMG liaison and a preferred place to start.

The CTSM and FATES users guide are excellent resources, 
- https://escomp.github.io/ctsm-docs/versions/master/html/index.html
- https://fates-users-guide.readthedocs.io/en/latest/

Like this discussion board the FATES community has also launched discussion pages on their github repository
- https://github.com/NGEET/fates/discussions

Finally, we're building a [library of tutorials](https://github.com/NCAR/CTSM-Tutorial-2022).  These are set up for the cloud, but also should work on an HPC system that has CESM ported and working.  in 2023 we hope to start developing more tutorials, like running FATES for single point and regional cases.

**Reply by wwieder** [reply]
Also, to get more information about CESM Land Model Working Group (LMWG) activities and workshops sign up for the listserv here https://mailman.cgd.ucar.edu/mailman/listinfo/ccsm-land

**Comment by andreaborrelli** [comment]
Dear all,

we are trying to implement CLM5.1 stand alone forced with GSWP3 repository forcings but we noticed a shift in time of diurnal cycle with respect to QIAN repository forcings. 

Namely selecting a specific lon, lat point (11E, 44N approximately) we got the maximum temperature is reached at different times (QIAN 13.30 as expected and GSWP3 10.30). 

Attached a screenshot of the two timeseries of the first two days of 2000 January for the selected point.

Is there a reason for that shift?

Thank you for your comments and suggestions.
Andrea
<img width="1264" alt="Screenshot 2023-06-09 at 11 09 09" src="https://github.com/ESCOMP/CTSM/assets/46671510/c6bd7a5f-b04b-46e2-8fc3-8c3ecb59e75c">
